packages/client/src/pages/RegisterAndLogin/RegisterAndLogin.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, useTheme } from '@mui/material';
+
+import dish from '../../assets/dish.jpg';
+import restaurant from '../../assets/restaurant.jpg';
+import { LoginForm, Question, RegisterForm, WelcomeMessage } from '../../components';
+import { routes } from '../../config';
+
+const userLogin = () => (
+  <>
+    <WelcomeMessage message='Witamy ponownie!' />
+    <LoginForm />
+    <Question question='Nie masz konta?' link={routes.userRegister} linkText='Zarejestruj się' />
+  </>
+);
+
+const partnerLogin = () => (
+  <>
+    <WelcomeMessage message='Witamy ponownie!' />
+    <LoginForm />
+    <Question question='Chcesz do nas dołączyć?' link={routes.partnerRegister} linkText='Zarejestruj się' />
+  </>
+);
+
+const userRegister = () => (
+  <>
+    <WelcomeMessage message='Stwórz konto' />
+    <RegisterForm userRole='User' />
+    <Question question='Masz już konto?' link={routes.userLogin} linkText='Zaloguj się' />
+  </>
+);
+
+const partnerRegister = () => (
+  <>
+    <WelcomeMessage message='Dołącz do naszych partnerów' />
+    <RegisterForm userRole='Partner' />
+    <Question question='Jesteś już w naszym programie?' link={routes.partnerLogin} linkText='Zaloguj się' />
+  </>
+);
+
+type RegisterAndLoginParams = {
+  formType: 'UserLogin' | 'UserRegister' | 'PartnerLogin' | 'PartnerRegister';
+};
+
+const RegisterAndLogin = ({ formType }: RegisterAndLoginParams) => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        minHeight: 'calc(100vh - 6rem)',
+        backgroundColor: theme.palette.background.default,
+        backgroundImage:
```

Comment:
I przy okazji zdjęcie ma jakieś 2MB, szkoda, że nie mamy nic do automatycznej optymalizacji zdjęć. To już zostawiam Tobie, jak masz czas to można ręcznie zmniejszyć i podmienić.

Reply:
Pobrałam mniejszy rozmiar zdjęć z Unsplasha, na laptopie wygląda to spoko, nie wiem niestety jak na większych ekranach bo nie mam teraz dostępu do mojego. Pozwoliłam sobie też zmienić zdjęcie w tle dla usera na trochę ciemniejsze, wtedy ciemny tryb ma sens 😀.